src/xcode/ENA/ENASecurity/Tests/ENASecurityTests/ECKeyPairGenerationTests.swift

```diff
@@ -0,0 +1,32 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import XCTest
+import CryptoSwift
+@testable import ENASecurity
+
+final class ECKeyPairGenerationTests: XCTestCase {
+    
+    func test_validPublicKeyBase64_isGenerated() {
+        let generationClass = ECKeyPairGeneration()
+        let generationOutput = generationClass.generatePrivateKey(with: "test")
+        guard let privateKey = generationOutput.0,
+              let publicKey = generationClass.generatePublicKey(from: privateKey) else {
+                  XCTFail()
```

Comment:
And Error message would be nice so we can see directly on the CI what went wrong here.